paper/paper.md

```diff
@@ -0,0 +1,122 @@
+   orcid: 0000-0001-5507-4304
+   affiliation: 6
+ - name: Kirstie Whitaker
+   orcid: 0000-0001-8498-4059
+   affiliation: 7
+affiliations:
+ - name: The Laboratory for Investigative Neurophysiology (The LINE), Department of Radiology and Department of Clinical Neurosciences, Lausanne, Switzerland; Center for Biomedical Imaging (CIBM), Lausanne, Switzerland
+   index: 1
+ - name: Centre for Neuroscience Studies, Queen's University, Kingston, Canada
+   index: 2
+ - name: Laboratory for Multimodal Neuroimaging, Philipps-University Marburg, Hesse, Germany
+   index: 3
+ - name: International Laboratory for Brain, Music and Sound Research, Université de Montréal & McGill University, Montréal, Canada
+   index: 4
+ - name: McConnell Brain Imaging Centre, Montréal Neurological Institute, McGill University, Montréal, Canada
+   index: 5
+ - name: Centre Leenaards de la Mémoire, Centre Hospitalier Universitaire Vaudois, Lausanne, Switzerland
+   index: 6
+ - name: Alan Turing Institute, University of Cambridge
+   index: 7
+date: 28 January 2019
+bibliography: paper.bib
+---
+
+
+# Summary
+
+A major advantage of magnetic resonance imaging (MRI) over other neuroimaging
+methods is its capability to noninvasively locate a region of interest (ROI) in
+the human brain. For example, using functional MRI, we are able to pinpoint
+where in the brain a certain cognitive process shows higher activation than
+usual. But just knowing the Cartesian coordinate of such a ROI is not useful if
+we cannot assign it a neuroanatomical label. For this reason, MRI images
+are usually normalized into a common template space [@fonov2011unbiased], where
+well-established atlases can be used to associate a given coordinate with the
+label of a brain region. Most major neuroimaging software packages provide some
+functionality to locate the main peaks of an ROI but this functionality is often restricted
+to a few atlases, frequently requires manual intervention, does not give the user much
+flexibility in the output creation process, and never considers the full extent
+of the ROI.
+
+To tackle those shortcomings, we created AtlasReader, a Python interface for
+generating coordinate tables and region labels from statistical MRI images. With
+AtlasReader, users can use any of the freely and publicly available neuroimaging
+atlases, without any restriction to their preferred software package, to create
+publication-ready output figures and tables that contain relevant information
+about the peaks and clusters extent of each ROI. To our knowledge, providing
+atlas information about the full extent of a cluster, i.e. over which atlas
+regions does a ROI extent, is a new feature that is not available in any other,
+comparable neuroimaging software package.
+
+Executing AtlasReader on an MRI image will lead to the creation of the following
+four outputs:
+
+1. An **overview figure** showing the ROIs throughout the whole brain at once
+   (see Figure 1).
```

Comment:
```suggestion
   (Figure 1).
```

Reply:
I got weirdly pedantic about the "see"s in the original version. These are not important edits from me 🤣